sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java

```diff
@@ -685,6 +685,7 @@ private FacilityReferenceDto getLabReference(LabMessageDto labMessageDto) {
 		if (!testReportDtos.isEmpty()) {
 			TestReportDto testReportDto = testReportDtos.get(0);
 			((ComboBox) sampleCreateComponent.getWrappedComponent().getField(PathogenTestDto.TEST_RESULT)).setValue(testReportDto.getTestResult());
+			((ComboBox) sampleCreateComponent.getWrappedComponent().getField(SampleDto.PATHOGEN_TEST_RESULT)).setValue(testReportDto.getTestResult());
```

Comment:
This does not align with the specification of the ticket (handling of several test reports). But I since the processing of multiple lab messages is currently not supported at all, this probably needs to go to another issue.

Reply:
I agree